src/diffusers/loaders.py

```diff
@@ -2390,7 +2390,7 @@ def unfuse_text_encoder_lora(text_encoder):
     def set_adapters_for_text_encoder(
```

Comment:
Nice! Going forward, let's make sure we always use strings for Transformers type hints